src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LOGO = "    ____        _        \n"
+            + "   |  _ \\ _   _| | _____ \n"
+            + "   | | | | | | | |/ / _ \\\n"
+            + "   | |_| | |_| |   <  __/\n"
+            + "   |____/ \\__,_|_|\\_\\___|\n";
+
+    public static final String BORDER_LINE = "------------------------------------------------";
+    
+    public static final int MAX_TASK = 100;
+    
+    public static final int CHARS_UNTIL_NUMBER = 5;
+
+    public static Task[] tasks;
+
+    public static void initializeTasks() {
+        tasks = new Task[MAX_TASK];
+    }
+
+    public static void greetUser() {
+        System.out.println("Hello from\n" + LOGO);
+        
+        System.out.println(BORDER_LINE + System.lineSeparator() 
+                + "    Hello!, I'm Duke" + System.lineSeparator()
+                + "    How can I help you?" + System.lineSeparator() 
+                + BORDER_LINE);
+    }
+
+    public static void goodbye() {
+        System.out.println(BORDER_LINE + System.lineSeparator() 
+                + "    Bye, see you again!" + System.lineSeparator()
+                + BORDER_LINE);
+    }
+    
+    public static String getCommand(String userInput) {
+        String[] words = userInput.split(" ");
+        return words[0];
+    }
+
+    public static void showTaskList() {
+        System.out.println(BORDER_LINE);
+        for (int i = 0; i < Task.getNumberOfTasks(); i++) {
+            System.out.println("    " + (i + 1) + "." + tasks[i]);
+        }
+        System.out.println(BORDER_LINE);
+    }
+
+    public static void markDone(String userInput) {
+        String extractedNumber = userInput.substring(CHARS_UNTIL_NUMBER);
+        int taskNumber = Integer.parseInt(extractedNumber) - 1;
+        tasks[taskNumber].markAsDone();
+        
+        System.out.println(BORDER_LINE + System.lineSeparator()
+                + "    The following task is now marked as done:" + System.lineSeparator()
+                + "      " + tasks[taskNumber] + System.lineSeparator() 
+                + BORDER_LINE);
+    }
+
+    public static void addTask(String userInput, String command) {
+        switch(command) {
```

Comment:
```suggestion
        switch (command) {
```
Missed out a whitespace for the switch condition.